ts/Series/Lollipop/LollipopSeriesOptions.d.ts

```diff
@@ -15,17 +15,23 @@
  * */
 
 import type LollipopSeries from './LollipopSeries';
-import type DumbbellSeriesOptions from '../Dumbbell/DumbbellSeriesOptions';
+import type ScatterSeriesOptions from '../Scatter/ScatterSeriesOptions';
 import type { SeriesStatesOptions } from '../../Core/Series/SeriesOptions';
+import type ColorString from '../../Core/Color/ColorString';
+import type ColorType from '../../Core/Color/ColorType';
 
 /* *
  *
  *  Declarations
  *
  * */
 
-export interface LollipopSeriesOptions extends DumbbellSeriesOptions {
-    lowColor?: undefined;
+export interface LollipopSeriesOptions extends ScatterSeriesOptions {
+    connectorColor?: ColorString;
+    connectorWidth?: number;
+    groupPadding?: number;
+    lowColor?: ColorType;
```

Comment:
Deprecate it and use the inherited [marker.fillColor](https://api.highcharts.com/highcharts/series.line.marker.fillColor) instead.